### PR TITLE
Add headers to post requests in fetcher.py

### PR DESCRIPTION
**Pull request by nineclicks**
#40

### CODE REVIEW

**Review by raishahnawaz — COMMENTED** [review]
These changes I have tested and worked for me.

**Comment by andormarkus** [comment]
Hi @konomae 
Please can you merge it?

**Comment by Alexandre-tKint** [comment]
@raishahnawaz could your merge please, thx!

File: lastpass/fetcher.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 http = requests
 
+headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.93 Safari/537.36'}
```
Comment (konomae) [inline lastpass/fetcher.py:23]: I will merge this.
Then change the User-Agent to another appropriate value. For example,`lastpass-python/0.3.1`.

**Review by konomae — COMMENTED** [review]
CI is failing. But I will merge and fix.

**Comment by konomae** [comment]
Thank you for your contribution! 🎉

File: lastpass/fetcher.py (hunk 1 above)
Comment (konomae) [reply lastpass/fetcher.py:23]: Fixed: https://github.com/konomae/lastpass-python/pull/42

**Pull request merged**